packages/rancher-prom2teams/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,56 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -2,9 +2,27 @@
+ # This is a YAML-formatted file.
+ # Declare variables to be passed into your templates.
+ 
++global:
++  cattle:
++    systemDefaultRegistry: ""
++  preInstallHook:
++    securityContext:
++      runAsNonRoot: true
++      runAsUser: 1000
```

Comment:
I'm not sure I agree with this being a global value.

IMO, instead of having it here we should set it as a value in the parent chart (`rancher-alerting-drivers`) and have the parent chart be responsible for installing the ConfigMaps once for all configured drivers that is referred to as an existing ConfigMap for all of the subcharts.

As a result, each subchart will need to introduce expose the following options instead of having ConfigMap-on-create logic:
```
configuration:
  # Defaults to using an existing ConfigMap in the deployment namespace called {{ .Chart.Name }}-prom2teams
  useExistingConfigMap: true
```

Reply:
We do not need to pass those values with the new approach for handling configmap.